src/deepforest/model.py

```diff
@@ -108,11 +108,16 @@ def __init__(self,
                  batch_size=4,
                  num_workers=0,
                  lr=0.0001,
+                 label_dict=None,
                  model=None):
         super().__init__()
         self.num_classes = num_classes
         self.num_workers = num_workers
-        self.numeric_to_label_dict = None
+        self.label_dict = label_dict
+        if label_dict is not None:
+            self.numeric_to_label_dict = {v: k for k, v in label_dict.items()}
+        else:
+            self.numeric_to_label_dict = None
```

Comment:
What are the consequences of leaving this as `None`? I assume this will get overwritten when a checkpoint is loaded per the PR description.

Reply:
a CropModel gets a label dict in one of two ways

1. As an input argument, this is mostly for testing and reloading checkpoints.
2. When you load a dataset for training.


https://github.com/weecology/DeepForest/blob/550e84288f96aaeefe50709cbd4546e8d8b337ea/src/deepforest/model.py#L177

I will add a ValueError if a user goes to predict but has not provided one of these two ways.